assets/dev/scss/global/global.scss

```diff
@@ -1,7 +1,6 @@
 @import "../../../../assets/dev/scss/helpers/variables";
 @import "../../../../assets/dev/scss/editor/breakpoints";
 @import "dialog";
-@import "aspect_ratio";
```

Comment:
why did you move it? can't we keep it here?

Reply:
In the past its loading twice in frontend.css & common.css because we want to use it also in the editor, now it's just for a BC in the front